controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -163,19 +178,22 @@ public ConfigurationFile(final File configurationDir, final String rawName, fina
      * @param useGit {@code true} if configuration is using Git to manage its history.
```

Comment:
Please add javadoc for the tmpDir param.

Reply:
javadoc added